server/seed.py

```diff
@@ -0,0 +1,156 @@
+            return "".join(random.choice(letters) for letter in range(length))
+    def generate_email():
+        return random_value("str", 10)+"@gmail.com"
+
+    users = []
+    profiles = []
+    recipes = []
+    is_chef = False
+    # Create how ever many user and profile objects the user defined
+    for i in range(num_of_instances):
+
+        # Create user
+        new_user = User.create(**{
+            'street_address': random_value('str', 16),
+            'city': random_value('str', 10),
+            'state_or_province': random_value('str', 15),
+            'country': random_value('str', 10),
+            'zip_code': "45698",
+            'username': random_value('str', 20),
+            'email': generate_email(),
+            # The first two args don't actually matter when is_password=True
+            'password': random_value("str", 15, is_password=True)
+            })
+        users.append(new_user)
+
+        # Create profile     
+        new_profile = Profile()
+        new_profile = new_profile.create(**{
+            'name': random_value('str', 15),
+            'is_chef': is_chef,
+            'about_me': random_value('str', 50),
+            'profile_image': random_value('str', 12),
+            'favourite_cuisine': random.choice(Recipe.get_cuisines()),
+            "location": random_value('str', 20),
+        })
+        profiles.append(new_profile)
+
+        # Assign the new profile to the new user
+        new_user.assign_one_to_one('profile', new_profile)
+        # Make some users normal, i.e no recipes
+        if not is_chef:
+            is_chef = True
+            continue
+        # Create 5 recipes for each user/profile
+        for j in range(0,6):
+            # For filter testing
+            if i % 3 == 1:
+                is_available = False
+            else:
+                is_available = True
+            # Generate random lists
+            random_items = [random.choice(string.ascii_letters) for x in range(6)]
+            random_ingredients = [random.choice(string.ascii_letters) for x in range(6)]
+            random_urls = [random.choice(string.ascii_letters) for x in range(6)]
+            new_recipe = Recipe.create(**{
+                'name': random_value('str', 25),
+                'description': random_value('str', 50),
+                'available': is_available,
+                'cuisine': random.choice(Recipe.get_cuisines()),
+                'price': random_value("str", 10, is_price=True),
+                'ingredients': ",".join(random_ingredients),
+                'required_items': ",".join(random_items),
+                'image_urls': ",".join(random_urls)
+            })
+            recipes.append(new_recipe)
+            # Add the new recipe to the One to Many field in the Profile model
+            new_profile.add_to_relationship('recipes', new_recipe)
+        if is_chef:
+            is_chef = False
+    write_fields_to_file(users, profiles, recipes)
```

Comment:
I think it would be better to just print the results rather than write them to a file